mentors/bash/m.json

```
@@ -1,4 +1,12 @@
 [
+  {
+    "github_username": "machuu",
+    "name": "Matt Hoover",
+    "link_text": "Personal Site",
+    "link_url": "https://hoover.ml",
+    "avatar_url": "http://gravatar.com/avatar/6dcd94787d387411653a88ac408c09dd.png",
+    "bio": "I use BASH daily at work and home, and I think it is a foundational skill for mastering any UNIX-like system."
+  }
```

you need a comma here
```suggestion
  },
```